### PR TITLE
adding some documentation

### PR DESCRIPTION
**Pull request by brotherthegrest**
adding some documentation to steps.go

### CODE REVIEW

**Comment by brotherthegrest** [comment]
bare minimum documentation in program.go

**Comment by brotherthegrest** [comment]
just adding some comments

**Comment by brotherthegrest** [comment]
fixing the conflict

File: cmd/program/program.go [hunk 1]
```diff
@@ -42,41 +43,40 @@ var (
...
 }
-
+// adding the go framework templates
 func (p *Project) createFrameworkMap() {
 	p.FrameworkMap["chi"] = Framework{
 		packageName: chiPackage,
 		templater:   tpl.ChiTemplates{},
 	}
-
+	// http/net
```
Comment (kevburnsjr) [inline cmd/program/program.go:59]: `net/http`

**Comment by brotherthegrest** [comment]
One mistake and you turn on me I had nothing to do with beego

**Comment by brotherthegrest** [comment]
No intentions of hacking or producing malware because that shit puts me in jail

**Pull request closed without merging**